pico/configdriven/processor/src/main/java/io/helidon/pico/configdriven/processor/ConfiguredByProcessorUtils.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.pico.configdriven.processor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import io.helidon.builder.AttributeVisitor;
+import io.helidon.builder.Builder;
+import io.helidon.builder.config.spi.ConfigBeanInfo;
+import io.helidon.common.config.Config;
+import io.helidon.common.types.TypeName;
+
+import static io.helidon.common.types.TypeNameDefault.create;
+
+final class ConfiguredByProcessorUtils {
+
+    static final String TAG_OVERRIDE_BEAN = "overrideBean";
+
+    private ConfiguredByProcessorUtils() {
+    }
+
+    static List<String> createExtraCodeGen(TypeName activatorImplTypeName,
```

Comment:
This method name does not make any sense, and there is no documentation here to explain what it does